python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -880,24 +886,38 @@ def _evaluate_auto_materialize_tick(
         else:
             sensor_tags = {SENSOR_NAME_TAG: sensor.name, **sensor.run_tags} if sensor else {}
 
-            run_requests, new_cursor, evaluations = AssetDaemonContext(
-                evaluation_id=evaluation_id,
-                asset_graph=asset_graph,
-                auto_materialize_asset_keys=auto_materialize_asset_keys,
-                instance=instance,
-                cursor=stored_cursor,
-                materialize_run_tags={
-                    **instance.auto_materialize_run_tags,
-                    **DagsterRun.tags_for_tick_id(
-                        str(tick.tick_id),
+            # experimental code path for evaluating scheduling in user space
+            if sensor and sensor.description == EXECUTABLE_AMP_SENSOR_DESC:
```

Comment:
pretty jank obviously, maybe OK for a prototype? Not something I would want to ship beyond that stage, both for back-compat and because its an actually useful field potentially

Reply:
yeah need the additional sensor type IMO